__tests__/__unit__/uss/__snapshots__/ZoweUSSNode.unit.test.ts.snap

```diff
@@ -62,26 +65,32 @@ ZoweUSSNode {
       },
       "shortLabel": "",
     },
-    "mProfileName": "aProfile",
+    "mProfileName": "sestest",
     "parentPath": null,
     "profile": Object {
       "failNotFound": false,
       "message": "",
-      "name": "aProfile",
-      "profile": Object {},
+      "name": "sestest",
+      "profile": Object {
+        "password": undefined,
+        "user": undefined,
+      },
       "type": "zosmf",
     },
     "profileName": "",
     "session": null,
     "shortLabel": "",
   },
-  "mProfileName": "aProfile",
+  "mProfileName": "sestest",
```

Comment:
Why change aProfile? Does this affect the other aProfiles out there?

Reply:
This is because the generator we made has mProfileName set to sestest, not aProfile. So it has been renamed here to match the generated profile.